packages/e2e/tests/giftcards/brandsConfiguration/giftcard.dropin.brandsConfiguration.test.js

```diff
@@ -0,0 +1,25 @@
+import DropinPage from '../../_models/Dropin.page';
+import GiftCardPage from '../../_models/GiftCardComponent.page';
+
+const dropinPage = new DropinPage({
+    components: {
+        giftcard: new GiftCardPage('.adyen-checkout__payment-method--giftcard')
```

Comment:
Minor thing - it seems weird to instantiate `GiftCardPage` inside `DropinPage` .

IMO it makes more sense to create a model like `GiftCard.component.js` which isn't a Page (does not extend from `BasePage`), but represents the Component. I did something similar [here](https://github.com/Adyen/adyen-web/pull/1448/files#diff-87008f4e9d7bbcdf1bb6aeed8beda9baf914bf3781a3b734e2a7109903ba6aeeR3-R4) for Address component.

We can brainstorm which way makes more sense and align that, so we can have a clear definition of page/model architecture for our E2E tests and write those following the same principles.

Reply:
I seem to recall it was your idea originally to do it like this...? (re. bancontact.visa.test.js using the CardCompoentPage)
But we can talk about it later.
And the point here is that the GiftCard comp that we need to test is the one that is found _inside_ the Dropin. When it's standalone it's already up to the merchant can display the name for the comp.

Reply:
> I seem to recall it was your idea originally to do it like this...? (re. bancontact.visa.test.js using the CardCompoentPage)
But we can talk about it later.

Yeap! My point here is that might make sense to create isolated `GiftCard.component.js` (without extending `BasePage`) and pass it to `DropinPage` as you did. Just trying to make clear the separation between page and reusable models/components 👍 

On `bancontact.visa.test.js` I missed this part of passing the CardCompoentPage to DropinPage


> And the point here is that the GiftCard comp that we need to test is the one that is found inside the Dropin. When it's standalone it's already up to the merchant to display the name for the comp.

Got it!